db/statusmapper.php

```diff
@@ -169,15 +169,17 @@ public function getUnindexed() {
 				$this->logger->
 					debug( 'expected string or instance of \OC\Files\Mount\Mount got ' . json_encode($mount) );
 			}
-			//only index local files for now
-			if ($storage->isLocal()) {
```

Comment:
how about `if ($storage && $storage->isLocal()) {`? Can you create a PR? Failing tests unrelated. Will merge when you update the PR :)